extra/xlunchr

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+# Copyright (C) 2018 xlunchr.
+#
+# xlunchr
+# a simple xlunch modernization
+# https://github.com/saymoncoppi/xlunchr
+#
+# This tool is based on original xlunch project
+# http://xlunch.org
+#
+# Author:     Saymon Coppi <saymoncoppi@gmail.com>
+# Maintainer: Saymon Coppi <saymoncoppi@gmail.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
This could maybe be rewritten? Right now it seems like it's a separate project, but this would merge it into xlunch.

Reply:
Maybe some additional info to say it was adapted from the original `xlunchr`?

Reply:
..or rename it to `xlunch-profile`, the nreomve the preceding `--` from the options, then you get usage like:

```
xlunch-profile desktop  # launch xlunch as desktop

xlunch-profile menu     # launch xlunch as menu

# etc...
```

Reply:
Why not just have two config files for xlunch and let the users modify those?

Reply:
Oh sorry, I thought you was the original creator of xlunchr. In that case we don't want to merge it into xlunch. It's a stand-alone project, and it might get merged in the future. But we'd want to do it properly with full integration.

Reply:
👍

Reply:
OK, I will remove and rename xlunchr, create my own repo for it, cheers